### PR TITLE
Fixed #48 && #49: Links and design

### LINKED ISSUE #48: Fix design
As everyone knows the site is a bit haywire right now we have to fix the design
The ways to do it - 
Apply padding where required
Fix colour combination

I am working on it

### PR DESCRIPTION
**Pull request by samyakbvs**

### CODE REVIEW

**Comment by samyakbvs** [comment]
I Won't be able to squash commits
screen shot 2017-12-25 at 9 58 14 pm
As they aren't appearing in my terminal

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyak-jainbvs-2014.fossasia.org/

File: _includes/header.html [hunk 1]
```diff
@@ -378,7 +378,7 @@
 	<input type="checkbox" id="show-menu" role="button" />
 	<ul id="menu">
 		<li><a href="index.html">Home</a></li>
-		<li><a href="http://fossasia.org/">FOSSASIA 2015</a></li>
+		<li><a href="https://2015.fossasia.org/">FOSSASIA 2015</a></li>
```
Comment (harshit98) [inline _includes/header.html]: Change 2015 -> 2014

**Comment by harshit98** [comment]
@samyakbvs Try to squash your commits from next time.

**Review by harshit98 — APPROVED** [review]

**Comment by anshumanv** [comment]
@samyakbvs you're always making escape from work by making some excuse(this is the last time, please approve, please merge merge), doing this will only lower your chances to win, @harshit98 please be a bit more strict with him.
samy

**Comment by samyakbvs** [comment]
@anshumanv I have no intentions of escaping from work and now you won't see things like this again 
Thanks!

**Comment by anshumanv** [comment]
>you won't see things like this again

Make sure this is **the** last time.

**Pull request merged**